definitions/marts/npq_separation_prep/ls_npqeyl_enrolments_v2.sqlx

```diff
@@ -0,0 +1,170 @@
+config {
+    type: "table",
+    assertions: {
+        uniqueKey: ["application_ecf_id"]

```

Comment:
Will we be using the ecf_id from applications in this mart? Surely it's preferable to switch to the new primary key for the id table?